api/registrations/views.py

```diff
@@ -799,8 +810,23 @@ def get_default_queryset(self):
     def get_queryset(self):
         return self.get_queryset_from_request()
 
+    def perform_create(self, serializer):
+        target = serializer.validated_data['target']
+        self.check_object_permissions(self.request, target)
+
+        if not target.provider.is_reviewed:
+            moderation_setup_url = absolute_reverse(
+                'providers:preprint-providers:preprint-provider-detail', kwargs={
+                    'provider_id': target.provider._id,
+                    'version': self.request.parser_context['kwargs']['version'],
+                },
+            )
+            raise Conflict(f'{target.provider.name } is an umoderated provider. If you are an admin, set up moderation by setting reviews_workflow at {moderation_setup_url}')
```

Comment:
I don't think this is true for Registries. I think only OSF Admins set up moderation for providers, we just allow them to add/remove moderators.

Reply:
Should we still perform this check?

Reply:
I think so, since we'll still have an unmoderated provider, but it could just say, "{target.provider.name} is an unmoderated provider."